modules/system/defaults/finder.nix

```
@@ -96,5 +96,13 @@ with lib;
       '';
     };
 
+    system.defaults.finder.NewWindowTarget = mkOption {
+      type = types.nullOr types.str;
```

```suggestion
      type = types.nullOr types.enum [ "PfHm" ... ];
```

If I understand correctly, only these values are possible:

https://gist.github.com/ChristopherA/98628f8cd00c94f11ee6035d53b0d3c6#file-macos-preferences-defaults-sh-L280-L286

---

Good call. I also added PfID which opens iCloud Drive